wiki/Storyboards/id.md

```diff
@@ -1,243 +1,54 @@
-__event, [...]
-```
-
-**Trigger Loops**:
-
-Trigger loops can be used to trigger animations based on play-time events. Although called loops, trigger loops only execute once when triggered.
-
-```
-_T,triggerName,start,end
-__event, [...]
-__event, [...]
-```
-
-**start**: When the trigger is valid **end** : When the trigger stops being valid
-
-Current triggers supported are: HitSoundClap HitSoundFinish HitSoundWhistle Passing (transition from fail state to pass state) Failing (transition from pass state to fail state)
-
-Trigger loops are zero-based similar to normal loops. If two overlap, the first will be halted and replaced by a new loop from the beginning. If they overlap any existing storyboarded events, they will not trigger until those transformations are no in effect.
-
-**Sound effects**
-
-    Sample,time,layer,"filepath",volume
-
-**time**: the time in milliseconds to start playing the sound effect
-
-**layer**:
-
--   Background = **0**
--   Fail = **1**
--   Pass = **2**
--   Foreground = **3**
-
-**filepath**: relative path of the sound file (should be in the same directory as the .osu file) *Note: file path must be in double quotes*
-
-**volume**: volume of the sound file. (1-100) Can be left off for 100.
-
-**Variables**
-
-Primitive support for variables is now available. You can define any number of variables by includin a \[Variables\] section in the .osb file (currently not supported in .osu-specific storyboarding).
-
-    [Variables]
-    $white=255,255,255
-
-Once you define variables, you can use them \*anywhere\* in the storyboard. This means you can have a variable containing as much as a whole line. Please don't make variable too small for the moment (ie. don't assign $var=1) because the editor is not smart enough to know where to put variables when it saves. All occurrences of the variables will currently be replaced at save time (so if you have one place you used $white and another you used 255,255,255 they will both become $white after a save in the editor).
+Storyboard Editor (SBE) terletak di tab Design (shorcut keyboard F2) dalam Beatmap Editor. Pada dasarnya, SBE menawarkan pengguna sebuah cara mudah untuk mengontrol pergerakan sprite sederhana, tetapi seringkali digunakan untuh hal-hal konyol dalam beatmap. Pengguna yang terampil dalam pembuatan storyboard seringkali menggunakan kode-kode hasil SBE, yang dinamakan Storyboard Scripting (SBS) karena SBS dapat menghasilkan kode yang tidak dapat dihasilkan SBE.
+
+
+Full Storyboard
+----------------
+
+Dengan adanya tab Design pada Beatmap Editor, pembuatan grafik visual dan animasi untuk beatmap menjadi lebih mudah dipelajari, berikut daftar lengkap beatmap berstatus ranked yang memiliki storyboard [panjang](https://osu.ppy.sh/p/beatmaplist?s=7&o=1&q=&r=0&g=0&la=0&sb=1). Laman ini hanya untuk menampilkan daftar beatmap dengan storyboard terpanjang saja, Jika Anda ingin daftar lebih banyak, silahkan kunjungi [daftar lengkap beatmap](/wiki/Storyboards/Maps).
```

Comment:
extend linking to
`[daftar lengkap beatmap berstatus ranked yang memiliki storyboard panjang](https://osu.ppy.sh/p/beatmaplist?s=7&o=1&q=&r=0&g=0&la=0&sb=1)`